ui/app/components/project-init-running.hbs

```diff
@@ -1,6 +1,8 @@
-<p>
-  <Spinner @size="20" />
+<p class="row-align">
+  <FlightIcon @name="loading" @size="16" />
+  &nbsp;
```

Comment:
**[question, non-blocking]**

I’m fine with these `&nbsp` entities, but I’m curious if there’s an equivalent to `grid-gap` for flexbox. I notice Tailwind has a [`gap`](https://tailwindcss.com/docs/gap) utility but I’m not sure how it works with flexbox.

Reply:
Added with flexbox `gap` in OIDC! Will keep the rest as-is to "match" the codebase